README.md

```diff
@@ -40,31 +40,30 @@ SecureNative::init("YOUR_API_KEY", new SecureNativeOptions());
 ## Event tracking
 
 ```
+$context = SecureNativeContext::fromRequest();
```

Comment:
It is required to import `SecureNativeContext` on top:
`use SecureNative\sdk\SecureNativeContext;`